Resources/Prototypes/Entities/Objects/Specific/Kitchen/foodcarts.yml

```diff
@@ -0,0 +1,163 @@
+- type: entity
+  name: Food Cart
+  id: FoodCartBase
+  abstract: true
+  parent: BaseStructureDynamic
+  description: A cart for food.
+  components:
+    - type: Sprite
+      netsync: false
+    - type: Rotatable
+    - type: InteractionOutline
+    - type: Fixtures
+      fixtures:
+        fix1:
+          shape:
+            !type:PhysShapeCircle
+            radius: 0.3
+          density: 250
```

Comment:
density can be lowered to make it a little easier to pull, since its implied to be on wheels

Reply:
What's a good value? Like 100? I'm not quite sure what to make it relative to.

Reply:
Went with 125, can tweak again if needed.